components/service/fretboard/src/main/java/mozilla/components/service/fretboard/Experiment.kt

```diff
@@ -16,6 +16,7 @@ data class Experiment(
     val match: Matcher? = null,
     val bucket: Bucket? = null,
     val lastModified: Long? = null,
+    val schema: Long? = null,
     val payload: ExperimentPayload? = null
 ) {
```

Comment:
We should add some kdoc to explain what schema is. I filed #524 to add kdoc in general here.